### DISCUSSION TITLE
Project aims

### DISCUSSION THREAD

**Discussion by adamski** [kunitoki/yup#5] [General]
I'm wondering about the general aims of this project. 

It seems there are two main issues the project is seeking to resolve:

1. **JUCE 8 License**: Creating an open and liberally licensed cross platform plugin and application development framework based on JUCE, in response to the JUCE 8 licence changes.
2. **UI development with JUCE**: Addressing one of the most time consuming aspects of JUCE development by integrating a modern high-performance UI and design framework (Rive).

**Comment by adamski** [comment]
I'm personally most interested in the Rive integration, but I also support an initiative for a community-led audio and plugin development framework.

**Comment by kunitoki** [comment]
The north star of this project could be: "Be an Open Source framework of choice for desktop and mobile application and plugin development, built and run by the community, that is able to target performant UIs rendering and allowing UI artists to leverage modern tools like Figma or Rive"
